src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx

```diff
@@ -42,6 +42,20 @@ import { Link } from "gatsby";
 
 ### How can you contribute to the Catalog?
  
+#### Create or Import a Design
+Begin by creating a new design from scratch or using a template:
+
+- **From Scratch**: Open the “Design” tab, select and arrange components on the canvas, and customize with connections, labels, and properties. When satisfied, save your design.
+- **From a Template**: Start from a pre-built template or clone an existing design from the Catalog. This allows you to build on established designs for a quicker start.
+
+Alternatively, import your own designs directly into the Catalog:
+
+- **Import a Design**: Access the Kanvas Designer and select the “Import Design” button in the left navigation panel. Upload a file or provide a URL for Docker Compose, Helm Charts, or Kubernetes Manifests. Kanvas will convert these into a usable design based on their configurations.
+
+#### Publish a Design
+Make your designs accessible to others by publishing them in the Catalog:
+
+- In Kanvas , locate the design you want to share and select “Publish.” Add any necessary details for the review process, such as relevant technologies, descriptions, and considerations. Once reviewed, your design becomes available to the broader Kanvas community.
```

Comment:
Here I believe `Sadam Mohamed` was updating this publish design documentation. maybe if he's done we can add that link. Or we need to tell user:
- a user needs to go to the design panel on [playground.meshery.io](http://playground.meshery.io/)
![image](https://github.com/user-attachments/assets/e5604a1a-4bfd-465e-855e-88f0b365b95c)
- when they hover over a design they'll find quick actions
![image](https://github.com/user-attachments/assets/dc2e0a78-7504-4a43-986d-7c64f6ae752b)
- then click on the info icon
![image](https://github.com/user-attachments/assets/ba909744-477d-4b50-86b4-eee3a94cc40c)
- They'll need to fill in the info inside this model and click on publish
![image](https://github.com/user-attachments/assets/e8f3703e-979a-4f4e-86e3-e893675bf4e1)